### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,8 +3,36 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T519278)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# DropDownBox - How to filter data of a nested widget when a user types text into the editor
+# DropDownBox for DevExtreme - How to filter data of a nested DataGrid
 
-In this example, dxDropDownBox enables end-users to filter list items dynamically based on the text typed into the editor's input box on the client side.
+This example demonstrates how to filter data of a DropDownBox nested DataGrid component.
```
Comment (dirk-pieterse) [inline Readme.md]: Can you please explain what a `DropDownBox nested DataGrid component` is? This is a noun string that is a bit confusing

File: Readme.md [hunk 2]
```diff
@@ -3,8 +3,36 @@
...
+
+![DropDownBox filtering](./dropdownbox-filtering.gif)
+
+Use [onInput](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onInput), [onOpened](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onOpened) and [onClosed](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onClosed) event handlers to manipulate and display data.
```
Comment (dirk-pieterse) [inline Readme.md]: Do not use `manipulate` because it has a negative meaning

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (vladaskorohodova) [reply Readme.md]: Maybe "embedded" would be better? A dropDownBox is a drop-down editor, and you can put whatever component you want into the drop-down field. In this example it's DataGrid.

File: Readme.md (hunk 2 above)
Comment (vladaskorohodova) [reply Readme.md]: I will change it for 'filter', cause it's more accurate.

File: Readme.md (hunk 1 above)
Comment (dirk-pieterse) [reply Readme.md]:
```suggestion
This example demonstrates how to filter the data of a DropDownBox embedded in a DataGrid component.
```

File: Readme.md (hunk 2 above)
Comment (dirk-pieterse) [reply Readme.md]: Ok

File: Readme.md [hunk 3]
```diff
@@ -3,8 +3,36 @@
...
 <!-- default badges end -->
-# DropDownBox - How to filter data of a nested widget when a user types text into the editor
+# DropDownBox for DevExtreme - How to filter data of a nested DataGrid
 
-In this example, dxDropDownBox enables end-users to filter list items dynamically based on the text typed into the editor's input box on the client side.
+This example demonstrates how to filter the data of a DropDownBox embedded in a DataGrid component.
+
+![DropDownBox filtering](./dropdownbox-filtering.gif)
+
+Use [onInput](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onInput), [onOpened](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onOpened) and [onClosed](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onClosed) event handlers to manipulate and display data.
```
Comment (vladaskorohodova) [inline Readme.md]:
```suggestion
Use [onInput](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onInput), [onOpened](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onOpened) and [onClosed](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#onClosed) event handlers to filter and display data.
```

**Pull request merged**